test/end_to_end/page.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env mocha */
+import { setup, teardown } from './helpers'
+
+describe('/terms', () => {
+  let gravity, browser
+
+  before(async () => {
+    ({ gravity, browser } = await setup())
+    gravity.get('/api/v1/page/:id', (req, res) => {
+      res.send({ name: 'Terms', content: 'Hello *World*' })
+    })
+  })
+
+  after(teardown)
```

Comment:
`before`/`after` runs before and after the whole test suite and `beforeEach`/`afterEach` runs between each test.